setup.py

```diff
@@ -150,6 +150,9 @@
 SetupDependency('nibabel', info.NIBABEL_MIN_VERSION,
                 req_type='install_requires',
                 heavy=False).check_fill(extra_setuptools_args)
+SetupDependency('h5py', info.H5PY_MIN_VERSION,
```

Comment:
I don't understand this. Why are we not using the requirements file to set up requirements that don't make crazy loops (such as numpy) ?

Reply:
I just went with what we have done for nibabel here. You may want to chat with @matthew-brett about this.  He most likely used this for a good reason.